### ISSUE TITLE
Wire is not voiceover friendly

### ISSUE THREAD

**Issue by rwaltr** [wireapp/wire-ios#687]
I am currently migrating my family to wire and my father is blind. Voiceover is not very verbose about what the buttons are and reads things hidden behind layers. Rendering it unusable. Is there anything possible to make it more friendly without compromising security?

**Comment by jenskasemets** [comment]
Hi @Blackphidora, thank you for your feedback. We have been improving accessibility for our app constantly, sadly there are still some major issues that block blind using it to full extend.
Luckily improving app's VoiceOver capabilities don't compromise any security, at least to our knowledge.

I really hope we can deliver you some great improvements in near future, stay tuned. Thank you for your patience and let us know if there are any specific use-cases that would help your father (calling, text messages, audio messages etc.?).

**Comment by mikeger** [comment]
@Blackphidora please find one of the improvements here https://github.com/wireapp/wire-ios/pull/717
